_layouts/page.html

```diff
@@ -44,6 +44,7 @@
             <li><a href="/get#managed-hosting">Hosting</a>
             <li><a href="/get#self-host">Download</a>
             <li><a href="/get#self-host">For Organizations</a>
+            <li><a href="https://github.com/sandstorm-io/sandstorm">Get the Source Code</a>
```

Comment:
This is pretty OK!

A suggestion: You can make this more concise by saying: "Source Code (GitHub)" rather than "Get the Source Code". That way, you avoid repeating the "Get" word that appears a few lines up, and you get to say both "GitHub" and "Source Code", which IMHO different sub-groups of the community will find meaningful, so it's nice to say both.

Curious what you think of that phrasing! Thanks for submitting this PR.